doc/user/content/get-started.md

```diff
@@ -217,3 +219,7 @@ Once you're done, don't forget to stop `curl` and `rm wikirecent`.
 [Next, see how Materialize can work as an entire microservices →](/docs/demos/microservice)
 
 {{</ cta >}}
+
+## <a name="webdemo"></a> Web Demo
```

Comment:
I'd rather make this a CTA like the link right above it and replace the existing link to this section with it.

Also, just FYI that all of the headers automatically generate an anchor tag, so you don't need to create your own. If you preview the page and click the header, it'll give you the appropriate hash.